webapp/src/components/CalendarPage.tsx

```diff
@@ -24,23 +32,25 @@ const CalendarPage = () => {
   if (!programs) {
     return null;
   }
+  const programEventsActivities: RBCEvent[] = programs[0].activities.map(
```

Comment:
We should figure out a way that allows us to conform our activities to `react-big-calendar`'s `Event` type so that we can use them directly instead of having to transform them. Or if we can't, some way to make our front-end code work better with the native types within `react-big-calendar`.